packages/nest/README.md

```diff
@@ -0,0 +1,125 @@
+<!-- markdownlint-disable MD033 -->
+<!-- x-hide-in-docs-start -->
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg" />
+    <img align="center" alt="OpenFeature Logo" src="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg" />
+  </picture>
+</p>
+
+<h2 align="center">OpenFeature Nest.js SDK</h2>
+
+<!-- x-hide-in-docs-end -->
+<!-- The 'github-badges' class is used in the docs -->
+<p align="center" class="github-badges">
+  <a href="https://github.com/open-feature/spec/releases/tag/v0.7.0">
+    <img alt="Specification" src="https://img.shields.io/static/v1?label=specification&message=v0.7.0&color=yellow&style=for-the-badge" />
+  </a>
+  <br/>
+  <a href="https://codecov.io/gh/open-feature/js-sdk">
+    <img alt="codecov" src="https://codecov.io/gh/open-feature/js-sdk/branch/main/graph/badge.svg?token=3DC5XOEHMY" />
+  </a>
+</p>
+<!-- x-hide-in-docs-start -->
+
+[OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API
+for feature flagging that works with your favorite feature flag management tool.
+
+🧪 This SDK is experimental.
+
+#### Here's a basic example of how to use the current API with flagd.
+
+#### Registering the Nest.js SDK module in the App Module:
+
+```ts
+import { Module } from '@nestjs/common';
+import { FlagdProvider } from '@openfeature/flagd-provider';
+import { OpenFeatureModule } from '@openfeature/nestjs-sdk';
+
+@Module({
+  imports: [
+    OpenFeatureModule.forRoot({
+      defaultProvider: new FlagdProvider({
+        host: 'localhost',
+        port: 8013,
+        tls: false,
+      }),
+      providers: {
+        differentServer: new FlagdProvider({
+          host: 'different-server.com',
+          port: 8013,
+          tls: false,
+        }),
+      },
```

Comment:
flagd is OK, but I think the in-memory provider would be a better example.

Reply:
Okay I can do this, just stole this from React 😄

Reply:
Changed